requirements.txt

```diff
@@ -3,13 +3,32 @@
 # process, which may cause wedges in the gate later.
 pbr!=2.1.0,>=2.0.0 # Apache-2.0
 pycodestyle>=2.5.0
-Pecan>=1.3.3
+Click>=7.0
```

Comment:
This appears to be introducing a bunch of requirements that aren't related to your configuration changes.